### PR TITLE
refactor(shared-data,api,components,app,pd): add gen3 pipettes

### PR DESCRIPTION
**Pull request by sfoster1**
Adds configuration, data, test exceptions, and frontend blocklists for OT3 pipettes.

These pipettes have the proper positioning and data for the OT3. They should not be used without that hardware and should not be loaded in OT-2 protocols (and can't physically be used on an OT2 anyway). This also means they should not show up in pipette selection dropdown menus in either PD or the app.

@Opentrons/qa The biggest thing we need to test here is that those dropdowns don't show the pipettes.

Do not merge until:
- [ ] dropdowns are tested
- [ ] 5.0.0 will no longer automatically pick up changes from `edge`

### CODE REVIEW

File: shared-data/js/constants.ts [hunk 1]
```diff
@@ -79,3 +79,5 @@ export const GEN_ONE_MULTI_PIPETTES = ['p10_multi', 'p50_multi', 'p300_multi']
 export const IDENTITY_VECTOR = { x: 0, y: 0, z: 0 }
 
 export const ROBOT_MODELS = ['OT-2 Standard', 'OT-3 Standard']
+
+export const OT3_PIPETTES = ['p300_single_gen3', 'p1000_single_gen3']
```
Comment (Laura-Danielle) [inline shared-data/js/constants.ts]: Why did we not include `p20_single_gen3` in this list?

**Review by Laura-Danielle — APPROVED** [review]
Had one comment, otherwise looks good.

File: shared-data/js/constants.ts (hunk 1 above)
Comment (sfoster1) [reply shared-data/js/constants.ts]: Accidentally. fixed!

**Review by ahiuchingau — APPROVED** [review]
Code looks good to me

**Pull request merged**